.zuul.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- job:
+    name: e2e-conformance
+    parent: openstack-access-base
+    description: |
+      Run tests of the configuration on test Kubernetes cluster using
+      [sonobuoy](https://sonobuoy.io/) with mode conformance meaning
+      it will test if the Kubernetes cluster is conformant to the CNCF.
+    pre-run: playbooks/dependencies.yaml
+    run: playbooks/e2e.yaml
+    cleanup-run: playbooks/cleanup.yaml  # executed also when the job is canceled
+    vars:
+      sonobouy_mode: conformance
+      wait_for_cluster: 600  # 10min
+    timeout: 10800  # 3h
+
+- job:
+    name: e2e-quick
+    parent: openstack-access-base
```

Comment:
This looks like the e2e-conformance test. You can use e2e-conformance as a parent here. Then you only have to overwrite the sonobouy_mode variable.

Reply:
Or you can use abstract job definitions.

Sample: https://github.com/osism/testbed/blob/main/.zuul.yaml#L30-L64

Reply:
Good idea, this will addressed within https://github.com/SovereignCloudStack/k8s-cluster-api-provider/issues/568